pages/02.general/11.https-filtering/01.https-filtering-known-issues/docs.en.md

```diff
@@ -38,9 +38,22 @@ AdGuard’s behavior depends on the platform. AdGuard for Windows uses Microsoft
 
 * Add a possibility to use CRLSets (an alternative to the OCSP by Google) to the validation library
 * Add the OCSP Must-Staple support to the validation library
```

Comment:
Must-Staple --> `Must-Staple`
Expect-Staple --> `Expect-Staple`
to keep consistency with RU